RELEASE_PROCESS.md

```diff
@@ -93,6 +189,22 @@ The git SHA of the build you are testing can be found in
   release, it should also be published to the Chef Security Announcements
   category.* Full details on the policy of making release announcements on
   Discourse can be found on the wiki under the Engineering section ->
-  Policy and Processes -> Release Announcements and Security Alerts
+  Policy and Processes -> Release Announcements and Security Alerts  
 
 Chef Infra Server is now released.
```

Comment:
Once the server omnibus release is complete; we should create an issue to update the version of Infra Server in Automate. This would be the last step in the process.

Reply:
Where do we create the issue?  Under chef-server repo?

Reply:
Let us start with creating it in the chef-server repo and take it from there.